notebooks/integrations/integration-git.ipynb

```diff
@@ -117,10 +117,13 @@
     }
    ],
```

Comment:
the notebook still has a screenshot of the old dashboard commit hash preview, could we update the screenshot with one of the new dashboard or just remove it all together?